### ISSUE TITLE
Regarding external libraries

### ISSUE THREAD

**Issue by romitheguru** [monikturing/turing-data-challenge#5]
`List of external libraries/packages used.`
To list down external libraries, should I just consider libraries that are not included by Python.
For example math, collections etc are Python's default libraries. Whereas Numpy, scikit-learn are needs to be installed first.

**Comment by monikturing** [comment]
Yes, we're only interested in the libraries that are not included by Python.
